### ISSUE TITLE
Unable to query licenses by product using an array

### ISSUE THREAD

**Issue by mcjackson18** [craftcms/commerce-digital-products#104]
### Description
Unable to query licenses by product using an array.

For example the following query:

`{% set licenses = craft.digitalProducts.licenses.owner(currentUser).product(['not', 'sku1', 'sku2']).with(['products']).all() %}`

Give this error:
`craft\digitalproducts\elements\db\LicenseQuery::product(): Argument #1 ($value) must be of type craft\digitalproducts\elements\Product|string|null, array given, called in /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php on line 1740`

### Additional info

- Craft version: Craft Pro 4.12.1
- PHP version: 8.2.13
- Database driver & version: MySQL 5.7.42
- Plugins & versions: Digital Products 3.2.3

**Comment by lukeholder** [comment]
You're correct, that param doesn't support arrays. You would need to do multiple queries for now.<br><br>Thanks for feature request, we will leave this open on the roadmap.
